docs/deploy-required-infrastructure.md

```diff
@@ -294,6 +294,133 @@ kubectl apply -k /opt/genestack/kustomize/ovn
 After running the setup, nodes will have the label `ovn.openstack.org/configured` with a date stamp when it was configured.
 If there's ever a need to reconfigure a node, simply remove the label and the DaemonSet will take care of it automatically.
 
+## Deploy PostgreSQL
```

Comment:
are we not able to run with the already available mariadb?

Reply:
I originally tried to get this working with the already available mariadb which at first glance in the chart appears to be possible. The more I dug into the chart, the more I found what appears to be a dependency on postgres. There might not be a hard dependency, but I wasn't able to get it to work after a number of attempts. Something to tinker with further most likely.

Ref: https://github.com/rackerlabs/genestack/pull/109/commits/98c3bbc0c583b409fc0b53e448a1ba43e5c10053#diff-34e7feda5857d85668a7788a5d69919a87f0856dda51a3a8c8bb48e2c91d199fR464 

Considering Gnocchi cites PostgreSQL [as the preferred indexer](https://gnocchi.osci.io/intro.html#indexer-driver) (likely the reason it's the default driver in the helm chart) - I opted to roll with it as the path of least resistance for now.

Reply:
I have no objections to the upstream postgres operator. That was what I looked into to begin with,  @talk2gw was looking into that separately from what I understand. I'm not sure if he's gotten that to work or not. The decision to use the existing postgres chart here isn't a final one, I simply need Gnocchi API running so I can start integrating Ceilometer. We can and should explore the operator iff we find we want/need postgres for Gnocchi moving forward.